### PR TITLE
Feature/Meilisearch Integrations std

### PR DESCRIPTION
**Pull request by brunoocasali**
Hey! @BlitzBanana, it's me again!

Since the latest release is out, I can finally give some attention to this long-awaited contribution!

Changes:

- I'm changing the README, adding the entire base of our integrations to look like our pattern. Besides appreciating the previous logo, I've asked our designers to build one for this integration, so I changed that as well & removed the old one.
- I've also started a CONTRIBUTING guideline, very similar to our style.

Basically, I'm making the integration look and feel at home 🥐🥖 
This is not the end. I will still add more stuff to the readme, like other search examples as seen [in](https://github.com/meilisearch/meilisearch-ruby#-getting-started) [others](https://github.com/meilisearch/meilisearch-js#-getting-started)

Related to: https://github.com/nutshell-lab/meilisearch-ex/issues/6

### CODE REVIEW

**Comment by BlitzBanana** [comment]
@brunoocasali Thank you for these changes.
I will review this PR asap.

**Comment by BlitzBanana** [comment]
@brunoocasali All good for me, do you want me to merge this before the following changes you mentioned ?

**Comment by brunoocasali** [comment]
Yes, you can merge. I'm not sure when I'll be able to continue the work!
